src/sage/groups/braid.py

```diff
@@ -1614,27 +1651,54 @@ def conjugating_braid(self, other):
 
         INPUT:
 
-        - ``other`` -- the other braid to look for conjugating braid
+        - ``other`` -- a braid in the same braid group as `self`.
+
+        OUTPUT:
+
+        A braid `d` such that `other` equals `d^-1 * self * d`.
 
         EXAMPLES::
 
             sage: B = BraidGroup(3)
             sage: a = B([2, 2, -1, -1])
             sage: b = B([2, 1, 2, 1])
             sage: c = b * a / b
-            sage: d = a.conjugating_braid(c)
-            sage: d * c / d == a
-            True
-            sage: d
+            sage: d1 = a.conjugating_braid(c)
+            sage: print(d1)
```

Comment:
You don't need the `print` here.
```suggestion
            sage: d1
```
Similar elsewhere.